Library/Homebrew/utils/github/api.rb

```diff
@@ -21,6 +21,7 @@ module GitHub
         #{ALL_SCOPES_URL}
       #{Utils::Shell.set_variable_in_profile("HOMEBREW_GITHUB_API_TOKEN", "your_token_here")}
   EOS
+  GITHUB_PAT_REGEX = /^(?:[a-f0-9]{40}|gp1_[A-Za-z0-9_]{40,255})$/.freeze
```

Comment:
I'm wondering if 255 includes the prefix or not.

Reply:
I just created a test token and it was 40 characters long overall, the blog post states "tokens of length upto 255" so I'm assuming it's the overall length.